boringtun/benches/crypto_benches/blake2s_benching.rs

```diff
@@ -1,23 +1,90 @@
-use boringtun::crypto::Blake2s;
+use blake2::digest::{FixedOutput, KeyInit};
+use blake2::{Blake2s256, Blake2sMac, Digest};
 use criterion::{BenchmarkId, Criterion, Throughput};
+use ring::rand::{SecureRandom, SystemRandom};
 
-pub fn bench_blake2s(c: &mut Criterion) {
-    let mut group = c.benchmark_group("blake2s");
+pub fn bench_blake2s_hash(c: &mut Criterion) {
+    let mut group = c.benchmark_group("blake2s_hash");
 
     group.sample_size(1000);
 
-    for size in [128, 1024] {
+    for size in [32, 64, 128] {
         group.throughput(Throughput::Bytes(size as u64));
 
-        group.bench_with_input(BenchmarkId::from_parameter(size), &size, |b, _| {
+        group.bench_with_input(BenchmarkId::new("blake2s_crate", size), &size, |b, _| {
```

Comment:
You don't need to do this, the group has only one member.

Reply:
These are leftovers from the intermediate commit where I was comparing the two. I'm leaving it in just in case we want to revert and do comparisons later, or if e.g. `ring` ever implements a Blake2 hash for comparison.

Reply:
See https://github.com/cloudflare/boringtun/pull/277/commits/f7f9f0a571b06819b997b566b7a2ad7212197bb8 for details.